src/container_host_image_startup_script.sh

```diff
@@ -20,6 +20,21 @@ sudo /gcsdk/google-cloud-sdk/install.sh --usage-reporting false --path-update tr
 sudo ln -s /gcsdk/google-cloud-sdk/bin/* /usr/bin
 EOF
 
+cat <<'EOF'
+#NVIDIA DRIVER
+sudo DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends nvidia-driver-450
+export distribution=$(. /etc/os-release;echo $ID$VERSION_ID)
+echo "trying to set var"
+echo $distribution
+curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | sudo gpg --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg \
+      && curl -s -L https://nvidia.github.io/libnvidia-container/experimental/$distribution/libnvidia-container.list | \
+         sed 's#deb https://#deb [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg] https://#g' | \
+         sudo tee /etc/apt/sources.list.d/nvidia-container-toolkit.list
+sudo apt-get update
+sudo apt-get install -y nvidia-docker2
+sudo systemctl restart docker
```

Comment:
Is it actually necessary to do this here?

Reply:
It was part of the nvidia instructions, but I didn't actually check if it worked without

Reply:
Testing in 3ac31cfceb7dc909f349f8070a4b37fe66f3cc2a; will revert if it breaks